tests/test_feature_extraction.py

```diff
@@ -0,0 +1,130 @@
+# coding=utf-8
+# Copyright 2022 the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from unittest import TestCase
+
+import habana_frameworks.torch as ht
+import pytest
+import torch
+import torch.nn.functional as F
+from transformers import AutoModel, AutoTokenizer
+
+from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
+
+
+adapt_transformers_to_gaudi()
+LATENCY_GTE_SMALL_BF16_GRAPH_BASELINE = 0.68102
```

Comment:
If this is the baseline for Gaudi2, I think there needs to be a check with environment variable GAUDI2_CI - see other tests for examples of this.

Reply:
This is the last update needed for my approval then I will handover to @regisss for review.

Reply:
@vidyasiv What is the standard process to obtain the latency measurements for non-GAUDI2_CI? I notice that some times are relaxed on GAUDI2_CI while others are tighter.

Reply:
When GAUDI2_CI is not set, the test is meant for Gaudi1. If you have access to Gaudi1 you can provide the numbers otherwise you can skip on your end. I am not aware of any standard for obtaining latency measurements otherwise. I assume they have to pass on the CI machines and the devs with access to push will update the values if needed.